include/application_defined_privileged_functions.h

```diff
@@ -92,5 +92,20 @@ void priv_eeprom_set_golden_info_(image_info app_info) {
     RESET_PRIVILEGE;
 }
 
+inline uint32_t priv_Fapi_BlockErase_(uint32_t ulAddr, uint32_t Size) {
```

Comment:
should it be priv_fapi_blockerase() as snake case? or is there a reason you named it like that?

Reply:
I kept the naming convention of the TI stuff for consistency. I don't like the look of it either